src/dmd/mars.d

```diff
@@ -1869,6 +1885,16 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
                         goto Lnoarg;
                     params.jsonfilename = p + 3 + (p[3] == '=');
                     break;
+                case 'q':
+                    if (!p[3])
+                        goto Lnoarg;
+                    if (p[3] != '=')
+                        goto Lerror;
+                    if (!p[4])
+                        goto Lnoarg;
+                    if (!parseJsonQuery(p + 4, &global.params.jsonQueryFlags))
```

Comment:
I see this is cumulative to jsonQueryFlags. Should it be? There's no way to turn off a flag with them being cumulative.

Reply:
All flags are off by default, you "turn then off" by not specifying them.  Are you saying that we should have a way to allow a user to turn it on in one argument, and they specify another argument to turn it off?